tools/Dashboard/DevHome.Dashboard/ViewModels/AddWidgetViewModel.cs

```diff
@@ -51,4 +59,16 @@ public void Clear()
         WidgetScreenshot = null;
         PinButtonVisibility = false;
     }
+
+    [RelayCommand]
+    internal async Task UpdateThemeAsync()

```

Comment:
super nit: `private`

Reply:
The MVVM toolkit generates the `RelayCommand` which calls those methods. XAML will use the (public) generate commands but won't be able to access the underlying private methods called. Similar to `ObservableProperty` attribute on private variables 🙂